### ISSUE TITLE
Clipboard support

### ISSUE THREAD

**Issue by cedrozor** [cedrozor/myrtille#3]
Myrtille doesn't support the clipboard and its associated features (copy&paste in particular).

Normally FreeRDP already support this functionality through virtual channels, but I haven't dug enough to figure it.

Please don't hesitate to add this feature to Myrtille, as it's a most wanted one. Thanks! :)

**Comment by borismod** [comment]
@cedrozor I'd love to take an issue and contribute. Just could not find the tests.

**Comment by cedrozor** [comment]
There are no tests actually (this may be just something next to do). Not easy, however, as there are not just simple methods to call with expected results; it implies display rendering and dynamic tweaking depending on the available bandwidth and latency...

I'm used to model and test driven developement (used UML, nUnit and mocks in my previouses works) but I also had to make a choice regarding my (limited) free time; so I decided to make real use cases tests, with the UI, than unit tests.

That said, I'll be glad to receive some help. I plan to upgrade to the latest version of freerdp in a near future, so it will be easier to bring new features such as clipboard support.

Thanks.
